recipe/gen_patch_json.py

```diff
@@ -850,6 +850,7 @@ def _gen_new_index(repodata, subdir):
             # ROOT requires vector-classes to be the exact same version as the one used for the build
             _replace_pin('vector-classes >=1.4.1,<1.5.0a0', 'vector-classes >=1.4.1,<1.4.2a0', deps, record)
 
+        _pin_stricter(fn, record, 'libunwind', 'x.x')
         _replace_pin('libunwind >=1.2.1,<1.3.0a0', 'libunwind >=1.2.1,<2.0.0a0', deps, record)
```

Comment:
Wouldn't we want to pin `libunwind<1.6` here as well, or am I misunderstanding the task?

Reply:
Yeah we probably want to do something else here. It's not clear to me atm what we want to do yet though

Reply:
Thanks, @mkitti, good catch. I removed the line and confirmed that the effect of that is that the pinning goes back to 1.3.0a0.